docs/TASKS.md

```diff
@@ -110,7 +110,51 @@ Process and Publish nodes in the workflow can also target remote snap nodes via
 
```

Comment:
Can't comment further but uppercase is missing on "**s**nap" [line 43](https://github.com/intelsdi-x/snap/pull/1269/files#diff-5710b592bec82bde9e7f0bae0a5bbe58R43).